event-app-api/merge_intervals/merge_intervals.js

```diff
@@ -0,0 +1,42 @@
+export function mergeIntervals(testEvents3) {
+  const merged_intervals = [testEvents3[0]];
+  for (let i = 1; i < testEvents3.length; i++) {
+    const currentEvent = testEvents3[i];
+    const lastMergedEvent = merged_intervals[merged_intervals.length - 1];
+
+    let currentEndTime = new Date(currentEvent.endtime);
+    let lastMergeEndtime = new Date(lastMergedEvent.endtime);
+    let currentStartTime = new Date(currentEvent.starttime);
```

Comment:
Do you need to do this conversion still? In your tests you're passing all events in as epoch time already and in the event_popup endpoint you're writing all the dates as epoch too.

Reply:
Before creating a different file to do the testing, I tried implementing it in the code and I had to change the startTime and endTime to epoch format, but I'll change that now.